dsgrid/tests/registration_models.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+
+from pydantic import validator
+
+from dsgrid.data_models import DSGBaseModel
+from dsgrid.utils.files import load_data
+
+
+class DatasetRegistrationModel(DSGBaseModel):
```

Comment:
Why is this in the `tests` folder?

Reply:
I expect that this will only be used in test code or by dsgrid developers that are building registries for test purposes.

Reply:
Can we add a comment at the top of expected use case for this script?